### PR TITLE
log only in debug, reorganize, potentially better page title

### PR DESCRIPTION
**Pull request by RichartE**

### CODE REVIEW

File: wikichange/src/content/enums.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const HighlightType = {
     TAGGING_WORD: 2,
 };
 
-const DEBUG = false;
+const DEBUG = true;
```
Comment (LuisaE) [inline wikichange/src/content/enums.js]: Let's merge with False

File: wikichange/src/content/globals.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+})();
+debug_console?.info(title);
+
+/**
+ * Get the Wikipedia page wikitext
+ *
+ * @return {string} wikitext of the article
+ */
+const getPageWikitext = async () => {
```
Comment (LuisaE) [inline wikichange/src/content/globals.js:58]: getPageWikiText

File: wikichange/src/content/markContent.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
-import { HighlightType, HIGHLIGHT_TYPE, DEBUG } from "./enums";
+import { HighlightType, HIGHLIGHT_TYPE } from "./enums";
 import { cleanText, escapeRegex, splitElementNode } from "./cleanText";
+import { debug_console } from "./globals";
```
Comment (LuisaE) [inline wikichange/src/content/markContent.js:3]: NIT: camelCase to match everything else.

**Review by LuisaE — CHANGES_REQUESTED** [review]

File: wikichange/src/content/enums.js (hunk 1 above)
Comment (sukritsangvong) [reply wikichange/src/content/enums.js]: \+

**Review by sukritsangvong — COMMENTED** [review]
Should be good after Luisa's comments r fixed

File: wikichange/src/content/globals.js (hunk 2 above)
Comment (RichartE) [reply wikichange/src/content/globals.js:58]: Wikitext is one word https://www.mediawiki.org/wiki/Wikitext

**Review by LuisaE — APPROVED** [review]
Looks good

**Pull request merged**